src/select-menu/src/OptionListGroupHeading.js

```diff
@@ -0,0 +1,102 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { TriangleIcon } from '../../icons'
+import { Pane } from '../../layers'
+import { Heading } from '../../typography'
+
+export default class OptionListGroupHeading extends PureComponent {
+  static height = 32
```

Comment:
What is this for?

Reply:
For virtualization we need to know the height up front.